plansys2_pddl_parser/src/plansys2_pddl_parser/Expression.cpp

```diff
@@ -63,34 +71,90 @@ Expression * createExpression( Stringreader & f, TokenStruct< std::string > & ts
 		f.next();
 		std::string s = f.getToken();
 		if ( s == "+" || s == "-" || s == "*" || s == "/" ) {
+   // It is a composite expression
 			CompositeExpression * ce = new CompositeExpression( s );
 			ce->parse( f, ts, d );
 			return ce;
-		}
-		else {
+		}	else {
+   // It is a function expression
```

Comment:
Unfortunately, most of the plansys2_pddl_parser files seem to use tabs instead of spaces for indentation. It's a bit hard to follow all the if, else if, and else statements in these changes as I believe you are using spaces. Could you use tabs to follow the original indentation scheme?

In the future, I would support replacing all the tabs in these files with spaces.

Reply:
If ok, I can change all the tabs to spaces since easier ;-) I was not noticing since this package is not subject to lint as for instance it is the case for the terminal.

Reply:
I'd be fine converting all tabs to spaces. Although, that might create a very large diff in this package. Perhaps it would be better to do that in a separate PR. We should see what @fmrico has to say about this.

Reply:
Yes, probably a separate PR is better. And, of, of course, spaces better tan tabs